diff --git a/node_modules/posthog-node/dist/entrypoints/index.node.d.ts b/node_modules/posthog-node/dist/entrypoints/index.node.d.ts
index e3803c8..6dc2dda 100644
--- a/node_modules/posthog-node/dist/entrypoints/index.node.d.ts
+++ b/node_modules/posthog-node/dist/entrypoints/index.node.d.ts
@@ -1,5 +1,9 @@
 export * from '../exports';
 import { PostHogBackendClient } from '../client';
+import error_tracking from "../extensions/error-tracking/index.mjs";
+
+export { error_tracking as ErrorTracking }
+
 export declare class PostHog extends PostHogBackendClient {
     getLibraryId(): string;
 }
diff --git a/node_modules/posthog-node/dist/entrypoints/index.node.mjs b/node_modules/posthog-node/dist/entrypoints/index.node.mjs
index 3b47634..87eca3a 100644
--- a/node_modules/posthog-node/dist/entrypoints/index.node.mjs
+++ b/node_modules/posthog-node/dist/entrypoints/index.node.mjs
@@ -4,6 +4,10 @@ import error_tracking from "../extensions/error-tracking/index.mjs";
 import { PostHogBackendClient } from "../client.mjs";
 import { ErrorTracking } from "@posthog/core";
 export * from "../exports.mjs";
+
+// Export because we want to add properties to alter the errors we send to posthog
+export { error_tracking as ErrorTracking }
+
 error_tracking.errorPropertiesBuilder = new ErrorTracking.ErrorPropertiesBuilder([
     new ErrorTracking.EventCoercer(),
     new ErrorTracking.ErrorCoercer(),
diff --git a/node_modules/posthog-node/src/.DS_Store b/node_modules/posthog-node/src/.DS_Store
new file mode 100644
index 0000000..d4971d7
Binary files /dev/null and b/node_modules/posthog-node/src/.DS_Store differ
diff --git a/node_modules/posthog-node/src/entrypoints/index.node.ts b/node_modules/posthog-node/src/entrypoints/index.node.ts
index a298059..db83cd4 100644
--- a/node_modules/posthog-node/src/entrypoints/index.node.ts
+++ b/node_modules/posthog-node/src/entrypoints/index.node.ts
@@ -7,6 +7,9 @@ import ErrorTracking from '../extensions/error-tracking'
 import { PostHogBackendClient } from '../client'
 import { ErrorTracking as CoreErrorTracking } from '@posthog/core'
 
+// Export because we want to add properties to alter the errors we send to posthog
+export { ErrorTracking }
+
 ErrorTracking.errorPropertiesBuilder = new CoreErrorTracking.ErrorPropertiesBuilder(
   [
     new CoreErrorTracking.EventCoercer(),
